webrender/src/frame.rs

```diff
@@ -52,6 +52,7 @@ pub struct Frame {
                                           AuxiliaryLists,
                                           BuildHasherDefault<FnvHasher>>,
     pub root_scroll_layer_id: Option<ScrollLayerId>,
+    pending_scroll_offsets: HashMap<PipelineId, HashMap<ServoScrollRootId, LayerPoint>>,
```

Comment:
wouldn't it be more efficient to have a single `HashMap<(PipelineId, ServoScrollRootId), LayerPoint>`?